website/content/docs/import/index.mdx

```diff
@@ -0,0 +1,122 @@
+---
+layout: docs
+page_title: Secrets import
+description: Secrets import allows you to safely onboard secrets from external sources into Vault KV for management.
+---
+
+
+# Secrets import
+
+<EnterpriseAlert product="vault" />
+
+@include 'alerts/beta.mdx'
```

Comment:
Same comment here, we might need guidance to create an `alpha` or `experimental` alert.

Reply:
I've added an experimental that I'll push up soon to be reviewed.